### PR TITLE
Changes to build on an M1 mac

### PR DESCRIPTION
**Pull request by leonid-s-usov**
It appears as if this change is pushing 2 commits, including the `50c6790`, but that's just because the commit hasn't been pushed to the master while it's currently the one recorded in the ceph/main as the submodule commit:

```
$git -P show origin/main -s
commit 38dea7dc2156bd003d2291e9f2d82688490beb53 (origin/main, main)
Merge: f82b9942d6d ad1202be685
Author: zdover23 <zac.dover@proton.me>
Date:   Wed Jul 26 09:07:40 2023 +1000

    Merge pull request #52630 from zdover23/wip-doc-2023-07-25-readmemd-1-of-x

    doc: update README.md

    Reviewed-by: Anthony D'Atri <anthony.datri@gmail.com>

$git ls-tree origin/main:src | grep seastar
160000 commit 50c6790df3db6ab38b5fea7e03494828dc2aafdc	seastar
```

This PR is supporting https://github.com/ceph/ceph/pull/52629

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "dpdk"]
 	path = dpdk
-	url = ../dpdk
+	url = https://github.com/ceph/dpdk
```
Comment (athanatos) [inline .gitmodules]: Why is this part necessary?

**Review by athanatos — CHANGES_REQUESTED** [review]
We generally get changes merged upstream -- the seastar upstream is generally very receptive.  I'd think the librt change would be pretty easy to convince them of.  I don't understand why the dpdk change is needed though.

File: .gitmodules (hunk 1 above)
Comment (leonid-s-usov) [reply .gitmodules]: this was needed for my local version since I didn't want to fork that submodule, too, just for the sake of the relative URL to work.
However, this PR itself doesn't require the change, so I can revert it

Comment (leonid-s-usov) [reply .gitmodules]: it's a separate commit, actually, so apparently I just pushed too much for the PR. I'll exclude that commit simply

Comment (leonid-s-usov) [reply .gitmodules]: done

**Comment by leonid-s-usov** [comment]
> We generally get changes merged upstream -- the seastar upstream is generally very receptive.

@athanatos my concern is whether it will be then acceptable to update the ceph main branch to whatever upstream head with this PR merged there. Or do you mean that once the change is accepted by the upstream we can then safely merge it locally over whatever version we currently have referenced?

**Comment by athanatos** [comment]
We tend to rebase the ceph seastar branch whenever seastar upstream has something we want -- this includes anytime we get something we want merged.  We should probably also do it every month or something, but it hasn't been a pressing issue.  Anyway, once your change merges it'll be easy to rebase/update our submodule.

**Comment by leonid-s-usov** [comment]
Following the investigation done in https://github.com/scylladb/seastar/pull/1806 this PR can be closed as incomplete
